app/scriptPlaylists.js

```diff
@@ -0,0 +1,72 @@
+var usersJSON = '[{"userName":"combs","email":"kathycombs@quailcom.com","password":"orkman5951"},{"userName":"whitley","email":"kathywhitley@quailcom.com","password":"inson2079"},{"userName":"fletcher","email":"kathyfletcher@quailcom.com","password":"rederick8363"}]';
+var jsonObject = JSON.parse(playlistsJSON);
+var usersObject = JSON.parse(usersJSON);
+
+// Array container for playlists
+var playlistsContainer = [];
+
+for(i = 0;i<jsonObject.length;i++){
+  playlistsContainer.push(new Playlist(i,jsonObject[i].title,jsonObject[i].description,jsonObject[i].image)); //  push new playlist into arrayContainer. !! description not yet implemented in JSON
+  playlistsContainer[i].setSongList(jsonObject[i].songs);   // setSongList because Playlist does not take songlist as constructor argument
+}
+
+  function getJson(list){
+      clearPlaylist();
+      var songsList = jsonObject[list].songs;
+     //alert(jsonObject[0].songs.length);
```

Comment:
Break this function into at least 4-5 functins: createSongTitle(title), createSongImage(image) etc. It's just too large at the moment.